packages/plugin-lighthouse/src/index.ts

```diff
@@ -1,5 +1,8 @@
+import { DEFAULT_FLAGS } from 'chrome-launcher/dist/flags.js';
 import { lighthousePlugin } from './lib/lighthouse-plugin';
 
+export const DEFAULT_CHROME_FLAGS = [...DEFAULT_FLAGS, '--headless'];
```

Comment:
I would also put it directly as default for the options. 

Using the plugin then could look like this:

```ts
lighthousePlugin('https://example.com', 
 // internally it will default to the following options if no chromeFlags options are given:
 // { chromeFlags: DEFAULT_CHROME_FLAGS }
);
```